src/pages/learn.tsx

```diff
@@ -109,6 +109,42 @@ const webinars = [
       ],
     },
   },
+  {
+    title: "Creation of Verifiable Documents (DID)",
+    description:
```

Comment:
```suggestion
    title: "Creation of Verifiable Documents (DNS-DID)",
    description:
```

- [v2](https://tradetrust-config.netlify.app/enums/node_modules__govtechsg_open_attestation_dist_types___generated___schema_2_0.identityprooftype)
- [v3](https://tradetrust-config.netlify.app/enums/node_modules__govtechsg_open_attestation_dist_types___generated___schema_3_0.identityprooftype)

method `DID` exists, however we are not recommending that fyi